src/Farmer/Arm/Dns.fs

```diff
@@ -51,25 +83,36 @@ module DnsRecords =
             .Map(fun r -> r + ".NSRecords")
             .WithOwner(sourceZoneResId)
 
+    let private ttlKey zoneType = match zoneType with | Public -> "TTL" | Private -> "ttl"
```

Comment:
I wonder if I can get rid of this by upgrading the schema for the public records?

Reply:
worth trying

Reply:
Unfortunately we're already using the latest schema for public dns records.